util/events/libuv/FileNo.c

```diff
@@ -93,9 +97,17 @@ static void incoming(uv_pipe_t* stream,
     } else {
         struct FileNo_pvt* fileno = Identity_check((struct FileNo_pvt*) stream->data);
         if (fileno->peer.accepted_fd != -1) {
+            int oldFd = fileno->peer.accepted_fd;
+            int newFd = 123;
```

Comment:
Maybe the dup() is also useless, dup() a file description only used for I/O redirecting or close the shared fd after fork from the parent process. The dup2() used here I can only see it is just for debugging.